easybuild/easyconfigs/n/NLMpy/NLMpy-0.1.5-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'NLMpy'
+version = '0.1.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.org/project/nlmpy'
+description = """NLMpy is a Python package for the creation of neutral landscape models that are widely used
+ in the modelling of ecological patterns and processes across landscapes."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['618eebc22a9fd8268893eaab14d40a2dd16939f31d59d00c46b6c1fc03e5b6d0']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+moduleclass = 'bio'
```

Comment:
```suggestion
moduleclass = 'geo'
```

Reply:
This was a hard call, it's really both bio and geo in a way. I'm just going to go with this so that we can get the PR accepted instead of any more bikeshedding.